src/items.js

```diff
@@ -0,0 +1,137 @@
+import { load, update } from './dynamic-list.js';
+
+const list = document.querySelector('ul');
+const input = document.querySelector('.input');
+const tasks = load();
+
+function creatitem(task) {
+  const newtask = {
+    description: task,
+    index: tasks.length + 1,
+    completed: false,
+  };
+
+  tasks.push(newtask);
+  update(tasks);
+}
+
+function changeStatus(checkbox, index) {
+  if (tasks[index].completed === true) {
+    tasks[index].completed = false;
+    update(tasks);
+  } else if (tasks[index].completed === false) {
+    tasks[index].completed = true;
+    update(tasks);
+  }
+}
+
+function addTodo(e) {
+  e.preventDefault();
+  // Create list
+  creatitem(input.value);
+  const newTodo = document.createElement('li');
+  newTodo.innerText = input.value;
+  newTodo.className = 'list-item';
+
+  list.appendChild(newTodo);
+  input.value = '';
+  // Create Completed Button
+  const checkbox = document.createElement('INPUT');
+  checkbox.setAttribute('type', 'checkbox');
+  checkbox.id = 'checkbox';
+  newTodo.appendChild(checkbox);
+
+  checkbox.addEventListener('change', () => {
+    changeStatus(checkbox, tasks.length - 1);
+  });
+
+  // // Create trash button
+  const trashButton = document.createElement('button');
+  trashButton.innerHTML = 'DELET';
+  trashButton.className = 'trash-btn';
+  newTodo.appendChild(trashButton);
+
+  // // Create edit button
+  const editButton = document.createElement('button');
+  editButton.innerHTML = 'edit';
+  editButton.className = 'edit-btn';
+  newTodo.appendChild(editButton);
+}
+
+function newTask() {
+  list.innerHTML = '';
+  tasks.forEach((item, i) => {
+    const li = document.createElement('li');
+    li.className = 'list-item';
+    li.innerText = tasks[i].description;
+    li.id = i;
+    list.appendChild(li);
+    const checkbox = document.createElement('INPUT');
+    checkbox.setAttribute('type', 'checkbox');
+    checkbox.checked = tasks[i].completed;
+    checkbox.id = 'checkbox';
+    li.appendChild(checkbox);
+    checkbox.addEventListener('change', () => {
+      changeStatus(checkbox, i);
+    });
+    const trashButton = document.createElement('button');
+    trashButton.innerText = 'DELET';
+    trashButton.className = 'trash-btn';
+    li.appendChild(trashButton);
+
+    // // Create edit button
+    const editButton = document.createElement('button');
+    editButton.innerText = 'edit';
+    editButton.className = 'edit-btn';
+    li.appendChild(editButton);
+
+    editButton.addEventListener('click', (event) => {
+      const newTask = prompt('enter');
+      const newId = event.target.parentNode.getAttribute('id');
+      tasks[newId].description = newTask;
+      update(tasks);
+      const anotherT = document.querySelector(`li[id="${newId}"]`);
+      anotherT.innerHTML = newTask;
+    });
+  });
+}
+
+function removeLocalTodos(tasks) {
```

Comment:
- [ ] When you remove an element, the index should be refreshed according to the list, always the first element should be one, the second two, please fix this issue. 🐶 
![image](https://user-images.githubusercontent.com/20563252/134753052-b5fd2a44-bfbf-47a5-abff-c74a81e6053c.png)